pkg/client-sdk/covenantless_client.go

```diff
@@ -34,6 +34,42 @@ import (
 	log "github.com/sirupsen/logrus"
 )
 
+// SettleOptions are only available for covenantless clients
+// it allows to customize the vtxo signing process
+type SettleOptions struct {
```

Comment:
```suggestion
type SignOptions struct {
```

maybe better?

Reply:
Personal preference for Settle cause "Sign" terms may also be associated with redeem tx (signing forfeit/redeem) 🤔 . `VtxoTreeOptions` ?